data/mods/Modular_Turrets/recipe.json

```diff
@@ -85,10 +89,11 @@
   {
     "type": "recipe",
     "result": "bot_EMP_hack",
+    "id_suffix": "repair",
     "copy-from": "bot_hack_base",
     "components": [
-      [ [ "bot_manhack", 1 ], [ "broken_manhack", 1 ], [ "broken_EMP_hack", 1 ] ],
-      [ [ "EMPbomb", 1 ] ],
+      [ [ "broken_manhack", 1 ], [ "broken_EMP_hack", 1 ] ],
+      [ [ "grenade_emp", 1 ] ],
```

Comment:
Whoops, my bad. Thanks for catching this.

Reply:
Yeah, I was thinking about what kinda UAV could fly the EMP bomb XD